doc/getting_started/installation_win.rst

```diff
@@ -72,7 +72,7 @@ environment for Windows. Follow the steps below to set it up:
    .. code-block:: console
 
       $ cd ~
-      $ git clone https://github.com/zephyrproject-rtos/zephyr.git
+      $ git clone --config core.autocrlf=false https://github.com/zephyrproject-rtos/zephyr.git
```

Comment:
Let's add that note to the text before this code block to explain why, something like this:

     #. From within the MSYS2 MSYS Shell, clone a copy of the Zephyr source into
        your home directory using Git.  (Some Zephyr tools require Unix-style line
        endings, so we'll configure Git for this repo to not do the automatic Unix/Windows
        line ending conversion (using ``--config core.autocrlf=false``).  Tools within the
        MSYS2 shell, such as vim, will behave properly
        but Windows tools such as notepad won't.)

Reply:
Done: https://github.com/zephyrproject-rtos/zephyr/pull/5604